gap/cliques.gi

```diff
@@ -519,16 +539,13 @@ function(arg)
       out := cliques;
     else
       # Act on the representatives to find all
-      orbits := [];
-      out := [];
+      orbits := HashMap();
       for c in cliques do
-        if not ForAny(orbits, x -> c in x) then
-          orb := Orb(G, c, OnSets);
-          Enumerate(orb);
-          Add(orbits, orb);
-          Append(out, orb);
+        if not c in orbits then
+          MultiOrbit(G, c, orbits);
```

Comment:
This line is a bit confusing, since `MultiOrbit` returns an orbit and its non-obvious that it changes the `orbits` hashmap in place (this is what its used here). Would it make sense to just inline the bit of the `MultiOrbit` code thats relevant to updating the `orbits` hashmap?

Reply:
I'm not sure it makes sense to inline the bit of code in `MultiOrbit`, it is used elsewhere, and then we'd have duplication of the method. Maybe I can rename the function to `AddOrbitToMap` or something?